src/components/tables/EntityTable.tsx

```diff
@@ -166,6 +167,12 @@ function EntityTable({
         }
     };
 
+    useEffectOnce(() => {
+        return () => {
+            return resetSelection();
```

Comment:
When the `EntityTable` component unmounts, reset the `setAll` component state value and remove the rows identified in the Select Table store.